cmd/kubeadm/app/master/manifests.go

```diff
@@ -68,12 +69,17 @@ func WriteStaticPodManifests(cfg *kubeadmapi.MasterConfiguration) error {
 		volumeMounts = append(volumeMounts, pkiVolumeMount())
 	}
 
+	k8sVersion, err := version.ParseSemantic(cfg.KubernetesVersion)
```

Comment:
I know on the command-line, we support strings like `latest` and `stable-1.6`. Are those guaranteed to be normalized to semver by this point?